composer.json

```diff
@@ -1,22 +1,26 @@
 {
     "name": "edujugon/push-notification",
     "description": "Laravel Package to send push notifications to Android and IOS devices. (GCM,FCM,APN)",
-    "keywords": ["Laravel","push","notification","gcm","apn","fcm","high","priority"],
+    "keywords": ["Laravel", "push", "notification", "gcm", "apn", "fcm", "high", "priority"],
     "license": "MIT",
+    "support": {
+        "issues": "https://github.com/Edujugon/PushNotification/issues",
+        "source": "https://github.com/Edujugon/PushNotification"
+    },
     "authors": [
         {
             "name": "Eduardo Marcos",
             "email": "edujugon@gmail.com"
         }
     ],
     "require": {
-        "php" : "~7.1.3||^7.2",
+        "php" : "~7.1.3|^7.2",
```

Comment:
@klimov-paul ,
Is not the single pipe "deprecated"?
I know It's kept for backwards compatibility but shouldn't we use the syntax that is [documented](https://getcomposer.org/doc/articles/versions.md#version-range)?

Reply:
Make sense - fixed.